packages/discord.js/src/structures/Widget.js

```diff
@@ -83,6 +83,16 @@ class Widget extends Base {
     this._patch(data);
     return this;
   }
+
+  /**
+   * Get Guild Widget Image
+   * @param {GuildWidgetStyle} [style=GuildWidgetStyle.Shield] The style for the widget image
+   * @returns {string}
+   */
+  imageURL(style = GuildWidgetStyle.Shield) {
```

Comment:
Why are we defaulting to this? I don't think we should have a default value for this.